Docs/source/usage/parameters.rst

```diff
@@ -1269,25 +1269,38 @@ Particle initialization
 * ``<species>.do_resampling`` (`0` or `1`) optional (default `0`)
     If `1` resampling is performed for this species. This means that the number of macroparticles
     will be reduced at specific timesteps while preserving the distribution function as much as
-    possible (in particular the weight of the remaining particles will be increased on average).
+    possible (details depend on the chosen resampling algorithm).
     This can be useful in situations with continuous creation of particles (e.g. with ionization
     or with QED effects). At least one resampling trigger (see below) must be specified to actually
     perform resampling.
 
 * ``<species>.resampling_algorithm`` (`string`) optional (default `leveling_thinning`)
-    The algorithm used for resampling. Currently there is only one option, which is already set by
-    default:
+    The algorithm used for resampling:
 
     * ``leveling_thinning`` This algorithm is defined in :cite:t:`param-MuravievCPC2021`.
-      It has two parameters:
+      It has one parameter:
 
         * ``<species>.resampling_algorithm_target_ratio`` (`float`) optional (default `1.5`)
             This **roughly** corresponds to the ratio between the number of particles before and
             after resampling.
 
-        * ``<species>.resampling_algorithm_min_ppc`` (`int`) optional (default `1`)
-            Resampling is not performed in cells with a number of macroparticles strictly smaller
-            than this parameter.
+    * ``grid_based`` The particles are sorted into phase space cells and merged.
+      It has three parameters:
+
+        * ``<species>.resampling_algorithm_delta_ur`` (`float`) optional (default `1e5`)
+            The width of momentum cells used in clustering particles, in m/s.
+
+        * ``<species>.resampling_algorithm_n_theta`` (`int`) optional (default `120`)
+            The number of cell divisions to use in the :math:`\theta` direction
+            when clustering the particle velocities.
+
+        * ``<species>.resampling_algorithm_n_phi`` (`int`) optional (default `60`)
+            The number of cell divisions to use in the :math:`\phi` direction
+            when clustering the particle velocities.
+
+* ``<species>.resampling_min_ppc`` (`int`) optional (default `1`)
```

Comment:
This name change is breaking the API. Is the name change needed?

Reply:
I will bring this up at the Tuesday meeting for general discussion. I think it is worth breaking the API for this since `resampling_min_ppc` is a general parameter for resampling, not an algorithmic specific option.

Reply:
This is the last issue - did anyone one else have an opinion? @axel ? @RemiLehe ?

Reply:
We have the `BackwardCompatibility` method - @roelof-groenewald can you check if the old parameter is set and if yes, error out with a clear migration guide.